grpc/src/bin/octree_server.rs

```diff
@@ -110,6 +113,41 @@ impl proto_grpc::Octree for OctreeService {
             .map_err(move |e| println!("failed to reply {:?}: {:?}", req, e));
         ctx.spawn(f)
     }
+
+    fn get_points_in_box(
+        &self,
+        ctx: RpcContext,
+        req: proto::GetPointsInBoxRequest,
+        resp: ServerStreamingSink<proto::GetPointsInBoxReply>,
+    ) {
+        let bounding_box = {
+            let bounding_box = req.bounding_box.clone().unwrap();
+            let min = bounding_box.min.unwrap();
+            let max = bounding_box.max.unwrap();
+            Aabb3::new(
+                Point3::new(min.x, min.y, min.z),
+                Point3::new(max.x, max.y, max.z),
+            )
+        };
+        let mut replies = Vec::new();
+        replies.push((proto::GetPointsInBoxReply::new(), WriteFlags::default()));
+        self.octree.points_in_box(&bounding_box).for_each(|p| {
+            let mut v = point_viewer::proto::Vector3f::new();
+            v.set_x(p.position.x);
+            v.set_y(p.position.y);
+            v.set_z(p.position.z);
+            replies.last_mut().unwrap().0.mut_points().push(v);
+            // Proto message must be below 4 MB.
+            // 240000 points take about 2.9 MB and proto requires about 1 MB of overhead for that.
```

Comment:
If we ever change the points proto, this heuristic will fail. Can we get this more principled somehow?